### ISSUE TITLE
Accessibility: Unable to tab through inbox

### ISSUE THREAD

**Issue by rolznz** [mailhog/MailHog#426]
I would like to use MailHog without the mouse, however tabbing doesn't actually allow me to select an inbox item to open.
